articles/101_build_tool.md

```diff
@@ -289,59 +291,142 @@ The following items are possible extension points to provide custom functionalit
+When the first draft of this article was written neither of the existing build tools supported the superset of features described in this article.
+There were multiple different paths possible to reach the goal of a universal build tool which fall into two categories:
 
 - One approach is to incrementally evolve one of the existing tools to satisfy the described goals.
 - Another approach would be to start "from scratch".
 
+Since then the new project `colcon` has been developed which covers most of the enumerated requirements and represents the second category.
+
 ### Evolve catkin_make, catkin_make_isolated, or ament_tools
 
 Since neither of these three build tools has the feature richness of `catkin_tools` it is considered strictly less useful to starting building upon one of these build tools.
+Therefore neither of these are being considered as a foundation for a universal build tool.
 
 ### Evolve catkin_tools
 
-Since `catkin_tools` is in many aspects the most complete build tool it should be the one being evolved.
+Since `catkin_tools` is in many aspects the most complete ROS build tool it should be the one being evolved.
 While `ament_tools` has a few features `catkin_tools` currently lacks (e.g. plain CMake support without a manifest, Windows support) the feature richness of `catkin_tools` makes it a better starting point.
 
-The following items are highlighting some of the necessary efforts (not a complete list):
+### Start "from scratch" / colcon
+
+Since the first draft of this article the `colcon` project has been developed with the goals and requirements of a universal build tool in mind.
+In its current form it is already able to build ROS 1 workspaces, ROS 2 workspaces, as well as Gazebo including its ignition dependencies.
+It uses Python 3.5+ and targets all platforms supported by ROS: Linux, macOS, and Windows.
+
+Since it hasn't been used by many people yet more advanced features like cross compilation, `DESTDIR`, etc. hasn't been tested (and will therefore likely not work yet).
+
+## Decision process
+
+For the decision process we are only considering the following two options based on the rational described above:
+
+A.  Use `catkin_tools` as a starting point
+B.  Use `colcon` as a starting point
+
+We acknowledge that if this topic would have been addressed earlier that some of the duplicate effort could have been avoided.
+When the work towards a universal build tool was suspended over a year ago it was a conscious decision based on available resources.
+Nevertheless we have to move forward with a decision in order to avoid further uncertainty and duplication.
+
+Both of the considered options have unique and valuable features and there are good arguments to build our future development on either of the two tools.
+Since both are written in Python either of the two tools could be "transformed" to cover the pros of the other one.
+So the two important criteria for the decision are:
```

Comment:
Are these the only criteria? If not, how did we down-select to these being the most important criteria?

Reply:
Imo the amount of time necessary to reach the stated goal is of highest importance. Simply because we are already very thin on resources and progress on feature development is rather slow. So any extra time spend on other items like this will slow our progress down. The second point is the quality of the end result since that will immediately effect the effort necessary to maintain and develop the software in the future. Therefore these are the "two important criteria".

There are certainly more but I didn't enumerate them here. Please feel free to add more from our internal documentation.